### DISCUSSION TITLE
Layout data in stats?

### DISCUSSION THREAD

**Discussion by BigBlueHat** [monkeytypegame/monkeytype#1802] [Ideas] [answered]
I'm trying various keyboard layouts--moving from QWERTY--and it'd be great to know which "run" used which layout. Right now, my (bad) Soul and Colemak-DH stats are mixed in together (and both abysmal...) while my QWERTY scores are obvious...because they're still higher.

It'd be super helpful to know which layout was used during a test.

Thanks for making MonkeyType!
🎩

**Comment by frogfrogfrogfrogfrog** [comment, marked as answer]
You can do this using tags. They're at the top of settings. You can create a tag and whenever your using QWERTY you can use a tag named QWERTY or something. Whenever, you use a different keyboard such as Colemak-DH you can simply switch to that tag and then you'll be able to seperate the different keyboard layouts.

**Comment by BigBlueHat** [comment]
That's what I ended up doing. Sadly means monkeytype itself can't find/show interesting stats about trending layout usage, but does give me personally way to keep track of stuff--when I remember to set them. 😉 

Thanks!
🎩

**Comment by kklocker** [comment]
While setting tags is a working solution, it is not super user friendly. 
I want to use the site for learning both a new layout,  **and** improving my qwerty typing.
Being such, I want to be able to switch contexts perhaps every other test or so, but the tags - way of doing this is a bit too cumbersome

**Reply by kklocker** [reply]
Perhaps an even better argument for this is that changing tags while using having layoutfluid enabled is really out of the question.

**Reply by kklocker** [reply]
> I dont understand how using tags hits the UX?
> 
> As for the 'profiles' - we have presets. You can save your current settings to a preset (including active tags) and load all settings with one click.
> 
> I find it unnecessary add layout tracking. This is why tags were created.

Alright knowing about presets changes my opinion of this :)

**Comment by takoyaro** [comment]
I'm currently learning Workman but I still practice my QWERTY typing.
Having to use tags really hits the UX for me.

I think it'd be nice to have different profiles under the account which activates different settings and layouts for a user.
For example, I don't need to see the visual keyboard when practicing QWERTY but I need it when I practice Workman.

Obviously, in the end, having stats per layout without the use of tags would be great.

**Reply by Miodec** [reply]
I dont understand how using tags hits the UX?

As for the 'profiles' - we have presets. You can save your current settings to a preset (including active tags) and load all settings with one click.

I find it unnecessary add layout tracking. This is why tags were created.

**Reply by Synaptic149** [reply]
I think what theyŕe trying to say is since the layout emulator is a setting you can set in Monkeytype, it should track it by itself without having to do extra steps.
